### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by mrego**
This will allow automatically labeling new issues depending on their type.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]:
```suggestion
name: Bug Report
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+about: Create a bug report to help us improve
+title: ''
+labels: C-untriaged
+assignees: ''
+
+---
+
+**Describe the bug:**
+A clear and concise description of what the bug is.
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]:
```suggestion
A clear and concise description of the bug.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+**To Reproduce:**
+Steps to reproduce the behavior.
+
+**Platform:** (optional)
+Specify the OS if relevant to the issue.
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]:
```suggestion
**Platform:**
Specify OS, distribution, and hardware platform.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]:
```suggestion
name: Feature Request
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+about: Suggest an idea or new feature request
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/feature_request.md:3]:
```suggestion
about: Suggest the implementation of a new feature
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+title: ''
+labels: B-feature-tracking, C-untriaged
+assignees: ''
+
+---
+
+**Describe the bug:**
+A clear and concise description of the feature request.
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]:
```suggestion
**Describe the new feature:**
A clear and concise description of the new feature with links to any relevant specifications.
```

File: .github/ISSUE_TEMPLATE/security-report.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+name: Security report
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/security-report.md:2]:
```suggestion
name: Security Vulernability
```

File: .github/ISSUE_TEMPLATE/security-report.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+about: Report a security vulnerability
+title: ''
+labels: A-security, C-untriaged
+assignees: ''
+
+---
+
+**Describe the bug:**
+A clear and concise description of what the bug is.
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/security-report.md:11]:
```suggestion
A clear and concise description of the vulernability
```

File: .github/ISSUE_TEMPLATE/security-report.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+**To Reproduce:**
+Steps to reproduce the behavior.
+
+**Platform:** (optional)
+Specify the OS if relevant to the issue.
```
Comment (mrobinson) [inline .github/ISSUE_TEMPLATE/security-report.md:17]:
```suggestion
Specify OS, distribution, and hardware platform.
```

**Review by mrobinson — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/security-report.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+name: Security Vulernability
+about: Report a security vulnerability
+title: ''
+labels: A-security, C-untriaged
+assignees: ''
+
+---
+
+**Describe the bug:**
+A clear and concise description of the vulernability
```
Comment (mrego) [inline .github/ISSUE_TEMPLATE/security-report.md:11]:
```suggestion
A clear and concise description of the vulnerability.
```

File: .github/ISSUE_TEMPLATE/security-report.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+
+**To Reproduce:**
+Steps to reproduce the behavior.
+
+**Platform:** (optional)
```
Comment (mrego) [inline .github/ISSUE_TEMPLATE/security-report.md:16]:
```suggestion
**Platform:**
```

**Comment by mrego** [comment]
Thanks for the review @mrobinson; I've applied your changes and fixed a typo.

**Review by mrobinson — APPROVED** [review]

**Pull request merged**